docs/INSTALL.K8S.md

```diff
@@ -0,0 +1,41 @@
+Installing Kubernetes
+=====================
```

Comment:
It seems unfortunate to have to maintain k8s instructions here.  Is there an official doc that can be referred to instead?  or in addition to this quick-start?